src/renderer/components/helpers/PseudoListItem.tsx

```diff
@@ -109,3 +110,25 @@ export const PseudoListItemAddContact = ({
     />
   )
 }
+
+export const PseudoListItemAddContactFromInviteLink = ({
+  inviteLink,
+  accountId,
+}: {
+  inviteLink: string
+  accountId: number
+}) => {
+  const tx = useTranslationFunction()
+  const processQr = useProcessQR()
+
+  return (
+    <PseudoListItem
+      id='newcontactfrominvitelink'
+      cutoff='+'
+      text={tx('menu_new_contact')}
+      onClick={() => processQr(accountId, inviteLink.trim())}
+    />
+    // Perhaps we could also parse username from the link
+    // and put it into `subText`.
```

Comment:
great idea

Reply:
I think rpc.checkQr should work for that, like it gets the name for the confirmation dialog when it asks you wether you want to chat with `<Name>`

Reply:
This could be improved in another PR. It would be nice to show both, the displayname & the mail address since the random testrun.org addresses are usually not known, only the displayname